x/chainlet/module.go

```diff
@@ -0,0 +1,164 @@
+package chainlet
```

Comment:
Both migrations packages should also be removed.

Reply:
They should be removed as they will never be used. Maybe you can leave v2 with an empty migration function in the repo for the future use.

Reply:
Done